### PR TITLE
Font typescript conversion

### PR DESCRIPTION
**Pull request by furkanarabaci**

### CODE REVIEW

File: ui/font/index.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { AbstractFont } from './font';
+
+const Font: typeof AbstractFont = require(`./font.${Device.deviceOS.toLowerCase()}`).default;
```
Comment (cenkce) [inline ui/font/index.ts:3]: Buraya type da eklemen lazim `type Font = AbstractFont `;

**Review by cenkce — CHANGES_REQUESTED** [review]

File: ui/font/index.ts (hunk 1 above)
Comment (furkanarabaci) [reply ui/font/index.ts:3]: Fixed

File: .types/missings-ios.d.ts [hunk 2]
```diff
@@ -1170,7 +1171,7 @@ declare class __SF_SMFCrypto {
 }
 
 declare class __SF_Database {
-  constructor(path: string | __SF_NSIndexPath);
+  constructor(path: string | string);
```
Comment (cenkce) [inline .types/missings-ios.d.ts:1174]: `string | string` kalmis

Comment (furkanarabaci) [reply .types/missings-ios.d.ts]: Fixed

**Pull request merged**